### PR TITLE
[info] Add info on Contributors Awards 2023 in README.md

### PR DESCRIPTION
**Pull request by jmMeessen**
Together with Alyssa Tong, I am proposing an update to the Jenkinsci organisation readme to announce the Jenkins Contributor Awards 2023 nomination and voting process.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -18,3 +18,21 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
...
+
+
+### Jenkins Contributor Awards 2023
+
+The annual Jenkins award  is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+
+Nomination categories are:
+- Most Valuable Jenkins Advocate → Nominate Someone
+- Most Valuable Jenkins Contributor → Nominate Someone
+- Jenkins Security MVP → Nominate Someone
```
Comment (lemeurherve) [inline profile/README.md:30]:
```suggestion
- Most Valuable Jenkins Advocate → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6035)
- Most Valuable Jenkins Contributor → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6033)
- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
```

File: profile/README.md [hunk 2]
```diff
@@ -18,3 +18,21 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
...
+
+Award timeline:
```
Comment (lemeurherve) [inline profile/README.md:32]:
```suggestion
Awards timeline:
```

**Review by lemeurherve — COMMENTED** [review]
LGTM, some suggestions.

File: profile/README.md [hunk 3]
```diff
@@ -18,3 +18,21 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
 
 The GitHub organization of the main Jenkins community including repositories of the [Jenkins core](https://github.com/jenkinsci/jenkins), plugins, libraries and developer tools is here:
 https://github.com/jenkinsci/
+
+
+### Jenkins Contributor Awards 2023
+
+The annual Jenkins award  is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
```
[the remark this answers is not in the document]
Comment (lemeurherve) [reply profile/README.md:25]: If the plural is unwanted, I'm suggesting only the suppression of the double space:
```suggestion
The annual Jenkins awards is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
```

File: profile/README.md [hunk 4]
```diff
@@ -18,3 +18,19 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
 
 The GitHub organization of the main Jenkins community including repositories of the [Jenkins core](https://github.com/jenkinsci/jenkins), plugins, libraries and developer tools is here:
 https://github.com/jenkinsci/
+### Jenkins Contributor Awards 2023
```
Comment (lemeurherve) [inline profile/README.md:21]:
```suggestion

### Jenkins Contributor Awards 2023
```
Bummer, my previous suggestion was to remove only one of the two empty lines, not both, not sure why it didn't work, here is the reinsertion of one empty line.

File: profile/README.md (hunk 2 above)
Comment (jmMeessen) [reply profile/README.md:30]: I will ask Alyssa who is author of the text. This requires a native english speaker.

File: profile/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jmMeessen) [reply profile/README.md:23]: I will ask Alyssa who is author of the text. This requires a native english speaker.

[the remark this answers is not in the document]
Comment (alyssat) [reply profile/README.md:24]: Thank you for catching that, pls keep the 's' in awards.

**Review by alyssat — APPROVED** [review]
This looks good to me. Thank you @jmMeessen for the PR.

**Review by lemeurherve — APPROVED** [review]

File: profile/README.md [hunk 5]
```diff
@@ -18,3 +18,20 @@ You can learn more about this subproject on the [website](https://www.jenkins.io
...
+The annual Jenkins awards is happening now! It recognizes contributions from individuals in the community and the progress made in the name of Jenkins.
+
+Nomination categories are:
+- Most Valuable Jenkins Advocate → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6035)
+- Most Valuable Jenkins Contributor → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6033)
+- Jenkins Security MVP → [Nominate Someone](https://github.com/jenkins-infra/jenkins.io/issues/6034)
+
+Awards timeline:
+- Nominations close: Friday, March 3
+- Voting opens: Wednesday, March 8
```
Comment (alyssat) [inline profile/README.md:33]: Voting opens: Wednesday, March 8 → [Vote Here](https://docs.google.com/forms/d/e/1FAIpQLScUL4GAL-6wOjHKbT86ptKSStnglKM9_MKTQXzjgwimCDEtGw/viewform)

Comment (alyssat) [reply profile/README.md:33]: Suggest to add link to voting form

**Review by alyssat — COMMENTED** [review]
Nomination is closed and voting is open. I'm adding the voting form.

File: profile/README.md (hunk 5 above)
Comment (jmMeessen) [reply profile/README.md:33]: Creating a dedicated PR for these proposed changes @alyssat

**Pull request merged**